### PR TITLE
some es5 to es6, coverage improvement and fix script commands

### PR DESCRIPTION
**Pull request by cgalvarez**
Some coverage improvement and some changes from es5 to es6.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -361,8 +356,8 @@ Why? When a browser opens the client side of your application, this package inte
 -   type: the type of report you want to create inside your `COVERAGE_APP_FOLDER`
 
     -   Default: `coverage`, used to dump the coverage object in a file because when there are several types of test, we want to merge results, and the server reloads between each one.
-    -   Allowed values: `cobertura`, `html`, `json`, `json-summary`, `lcov`, `teamcity`, `text`, `text-lcov`, `text-summary`, `lcovonly`, `coverage`, `remap`
-    -   **Not working values:** `clover`, `cobertura`, `lcov`, `text`, `text-lcov`, PR welcome
+    -   Allowed values: `cobertura`, `coverage`, `html`, `json`, `json-summary`, `lcov`, `lcovonly`, `remap`, `teamcity`, `text`, `text-lcov`, `text-summary`
```
Comment (serut) [inline README.md]: remove teamcity

Comment (cgalvarez) [inline README.md]: The "not working" values are listed in "Allowed values" too. I should remove then `cobertura`, `text-lcov`, `lcov`, `text`.

File: server/handlers.js [hunk 2]
```diff
@@ -112,18 +121,12 @@ instrumentClientJs = function (params, req, res, next) {
   }
 };
 
-
+let noop = function () {};
 export default Handlers = {
```
Comment (serut) [inline server/handlers.js]: sadness... You optimized some legacy code. Handlers is not exposed anymore if `Conf.IS_ACTIVE === false`, so it does not require this mechanism

Comment (cgalvarez) [inline server/handlers.js]: So I can remove all the export...

Comment (serut) [inline server/handlers.js]: 👍

File: README.md (hunk 1 above)
Comment (serut) [inline README.md]: My bad. As you want

**Comment by serut** [comment]
All right for me.

**Pull request merged**